bigquery/cloud-client/async_query.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Command-line application to perform asynchronous queries in BigQuery.
+
```

Comment:
It would be nice in this application and the sync query one to show an example usage, e.g.:

```python sync_query.py "SELECT corpus FROM `publicdata.samples.shakespeare` GROUP BY corpus"````